docs/hosting/configuration/task-runners.md

```diff
@@ -28,25 +30,54 @@ You can use task runners in two different modes: internal and external.
 
 ### Internal mode
 
-In internal mode, the n8n instance launches the task runner as a child process. The n8n process monitors and manages the life cycle of the task runner. The task runner process shares the same `uid` and `gid` as n8n.
+In internal mode, the n8n instance launches the task runner as a child process. The n8n process monitors and manages the life cycle of the task runner. The task runner process shares the same `uid` and `gid` as n8n. This is **not** recommended for production.
 
 ### External mode
 
-In external mode, an external orchestrator (for example, Kubernetes) launches the task runner instead of n8n. Typically, this means you would configure the task runner to run as a side-car container next to n8n.
+In external mode, a [Go launcher](https://github.com/n8n-io/task-runner-launcher) launches task runners on demand and manages their lifecycle. Typically, this means that next to n8n you add a sidecar container running the [`n8nio/runners`](https://hub.docker.com/r/n8nio/runners) image containing the launcher, the JS task runner and the Python task runner. This sidecar container is fully isolated from the n8n instance.
```

Comment:
Is golang as the launcher technology important to mention here? I think we should just talk of the task-runner-launcher application without really mentioning more of how it's made.

Reply:
> fully isolated

they can of course communicate with each other over the network, perhaps we should just talk about it being independent from the n8n instance 🤔